index.html

```diff
@@ -1,290 +1,227 @@
-<!DOCTYPE html>
+---
```

Comment:
I think we should revert the changes to the homepage in this branch

Reply:
What does this mean?